webplugin/js/app/kommunicate.js

```diff
@@ -518,9 +518,6 @@ $applozic.extend(true,Kommunicate,{
     // check if the message needs to be processed by addMessage
     visibleMessage: function(msg){
         if(!msg) return false;
-        if(msg.metadata && msg.metadata.feedback){
```

Comment:
what was the purpose of this code which is removed? is it not required now?

Reply:
Previously, whenever there was a message containing feedback it was ignored.

Now it is not ignored as a user may give multiple feedbacks (CSAT ratings) and is shown along with the messages as is shown in the dashboard. 

![Screenshot 2020-12-29 at 4 50 35 PM](https://user-images.githubusercontent.com/19753380/103280273-fc154680-49f5-11eb-897e-ece9dfc9ebf9.png)